File: social-service/src/test/java/com/eirsteir/coffeewithme/SocialServiceApplicationTests.java
package com.eirsteir.coffeewithme;

import com.eirsteir.coffeewithme.social.SocialServiceApplication;
import com.eirsteir.coffeewithme.testconfig.EventuateTestConfig;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@ActiveProfiles("test")
@SpringBootTest(classes = {SocialServiceApplication.class, EventuateTestConfig.class})
class SocialServiceApplicationTests {

    @Test
    void contextLoads() {
    }

}

File: social-service/src/test/kotlin/com/eirsteir/coffeewithme/SocialServiceApplicationTests.kt
package com.eirsteir.coffeewithme

import com.eirsteir.coffeewithme.config.EventuateTestConfig
import com.eirsteir.coffeewithme.social.SocialServiceApplication
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles

@ActiveProfiles("test")
@SpringBootTest(classes = [SocialServiceApplication::class, EventuateTestConfig::class])
internal class SocialServiceApplicationTests {
    @Test
    fun contextLoads() {
    }
}